src/components/TabbedQueryEditor.tsx

```diff
@@ -117,7 +117,7 @@ export const TabbedQueryEditor = ({ query, onChange, onRunQuery, fieldsTab, expe
                 margin-bottom: ${theme.spacing.sm};
               `}
             >
-              {({ width }) => (
+              {({ width }: { width: number }) => (
```

Comment:
This change shouldn't be needed if you remove the types/autosizer lib and bump typescript to 5.2.2